erpnext/accounts/report/accounts_receivable_summary/accounts_receivable_summary.js

```diff
@@ -106,6 +106,11 @@ frappe.query_reports["Accounts Receivable Summary"] = {
 			"label": __("Sales Person"),
 			"fieldtype": "Link",
 			"options": "Sales Person"
+		},
+		{
```

Comment:
@nabinhait Is this PR complete? There is no effect of the filter on report

Filter Applied:
![image](https://user-images.githubusercontent.com/42651287/70114829-d1867c00-1683-11ea-994c-005b35f19574.png)

Filter Not Applied:
![image](https://user-images.githubusercontent.com/42651287/70114855-ed8a1d80-1683-11ea-8650-214844b71a1d.png)

A Due date column should also be added for the report to make sense if the filter is ticked

Reply:
Yes, the PR is complete.
We can't add due date column, as it is a summary report. Basically, if we check "Based on Payment Terms" filter on summary report, we get ageing data as per payment terms due date. Nothing else are changed.